src/Exceptions/DuplicatedIndexException.php

```diff
@@ -1,17 +0,0 @@
-<?php
-
-namespace Spatie\Enum\Exceptions;
-
-use InvalidArgumentException;
-
-class DuplicatedIndexException extends InvalidArgumentException
```

Comment:
I think that we still need this Exception and should also throw it because we still allow custom maps which aren't enforced to be unique.
But rename it to `DuplicateValueException`.

Reply:
I agree!